nextcloud_scan.py

```diff
@@ -66,42 +66,58 @@ def getResultJson(self):
         result = self.doScan()
 
         result['vulnerabilitiesCount'] = len(result['vulnerabilities'])
-        result['hardeningsMissing'] = sum( 1 for isHardened in result['hardenings'].values() if isHardened == False )
-        result['headersMissing'] = sum( 1 for headerPresent in result['setup']['headers'].values() if headerPresent == False )
+        result['hardeningsMissing'] = sum(1 for isHardened in result['hardenings'].values() if isHardened is False)
+        result['headersMissing'] = sum(1 for headerPresent in result['setup']['headers'].values() if headerPresent is False)
 
-        return json.dumps(result, separators=(',',':'))
+        return json.dumps(result, separators=(',', ':'))
+
+    def getFormattedResultJson(self):
+        result = self.doScan()
+
+        result['vulnerabilitiesCount'] = len(result['vulnerabilities'])
+        result['hardeningsMissing'] = sum(1 for isHardened in result['hardenings'].values() if isHardened is False)
+        result['headersMissing'] = sum(1 for headerPresent in result['setup']['headers'].values() if headerPresent is False)
+
+        # Hier erfolgt die formatierte Ausgabe mit Einrückungen
+        return json.dumps(result, indent=4, separators=(',', ':'))
```

Comment:
This is duplication of the `getResultJson` method. The only difference is adding the `indent=4` argument to `json.dumps()`. I'm not against adding a verbose flag, but please refactor it so this method doesn't duplicate most of its code.